src/sms-password/templates/user_management/sms-password/reset_form.html

```diff
@@ -6,35 +6,36 @@
     <p translate>You've successfully changed your password.</p>
 </div>
 
-<div class="form-group" ng-class="{'has-error': errors.phone_number}">
+<div class="form-group" ng-class="{'has-error': errors.phone_number || errors.non_field_errors}">
     <label for="phone_number" ng-show="fields.phone_number.label" ng-bind="fields.phone_number.label"></label>
     <input type="text" ng-model="data.phone_number" class="form-control" id="phone_number" autofocus>
     <div class="help-block" ng-show="errors.phone_number" ng-bind="errors.phone_number"></div>
+    <div class="help-block" ng-if="errors.non_field_errors" ng-bind="errors.non_field_errors"></div>
 </div>
 
 <button type="submit" class="btn btn-default"  ng-click="resetRequest()" translate>Request code</button>
 
+<div ng-show="requested">
```

Comment:
Why are all the fields being hidden until `requested` is true? The commit messages don't explain this.

What about when `requested === false`? What's shown then? Do you have an empty form?